docs/application/tizen-studio/release-notes/4-1-release-notes.md

```diff
@@ -13,8 +13,8 @@
 
 ### Deprecated
    - Tizen RT IDE has been deprecated from Tizen Studio 4.1.
+   - Tizen CLI does not support C# projects from Tizen 5.5 onwards.
```

Comment:
I understood what you said. 
Because "onwards" was not our style to write technical docs, I thought "5.5 onwards" is awkward.
However, I found out a similar style phrase many times when I googled.
Thank you for your update.